src/swarmus_ros_simulation/launch/hive_mind_connector_dummy.launch

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<launch>
+
+    <node pkg="swarmus_example_pkg" type="hive_mind_connector_dummy" name="hive_mind_connector_dummy"/>
+    <node pkg="hive_mind" type="hive_mind" name="hive_mind"/>
+    
```

Comment:
If we need to see the console output for the ros node, simply add output="screen" to the node parameters.